psalm-baseline.xml

```diff
@@ -1593,6 +1593,9 @@
       <code>$container</code>
       <code>$container</code>
     </MixedAssignment>
+    <MixedInferredReturnType occurrences="1">
+      <code>($name is class-string ? T : callable)</code>
```

Comment:
I think this should be `: HelperInterface|callable`, not just `callable`

Reply:
Careful: `callable` does not mean `->__invoke()`, since an `array` is a valid callable sometimes :-) 

If you do `->get('foo')`, it may also be a helper